oem/ibm/libpldmresponder/file_io.cpp

```diff
@@ -97,7 +97,16 @@ int DMA::transferHostDataToSocket(int fd, uint32_t length, uint64_t address)
         std::cerr
             << "transferHostDataToSocket : Failed to execute the DMA operation, RC="
             << rc << " ADDRESS=" << address << " LENGTH=" << length << "\n";
-        munmap(vgaMemDump, pageAlignedLength);
+        if (rc != -EINTR)
+        {
+            munmap(vgaMemDump, pageAlignedLength);
+        }
+        else
+        {
+            std::cerr
+                << "Received interrupt during dump DMA transfer. Skipping Unmap"
+                << std::endl;
```

Comment:
can you add a function name ? as well so that we know this came from what place ?

Reply:
apart from this the changes look good.

Reply:
done